ambry-account/src/main/java/com/github/ambry/account/mysql/MySqlAccountStoreFactory.java

```diff
@@ -69,4 +70,21 @@ public MySqlAccountStore getMySqlAccountStore() throws SQLException {
       throw e;
     }
   }
+
+  /**
+   * Returns an instance of the {@link MySqlAccountStore} that the factory generates.
+   * @return an instance of {@link MySqlAccountStore} generated by this factory.
+   * @throws SQLException
+   */
+  public MySqlAccountStore getMySqlAccountStoreNew() throws SQLException {
```

Comment:
Parameterize getMySqlAccountStore() and call it for multiple times for the existing and new DBs?